### PR TITLE
Updated functions in urls.py both in the webapp folder and templates.…

### PR DESCRIPTION
**Pull request by Mhachem99**
… Created login.html page
 and updated some functions in other files

### CODE REVIEW

File: webapp/main/urls.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-from django.urls import path
```
Comment (AhmedMinhajEmon) [inline webapp/main/urls.py:1]: Why u remove django.urls??

**Review by AhmedMinhajEmon — APPROVED** [review]
Looks good

File: webapp/main/urls.py (hunk 1 above)
Comment (AafnanM) [inline webapp/main/urls.py:1]: does the server still run correctly without .url? The server wouldn't run for me until I added that

**Review by AafnanM — COMMENTED** [review]
Looks good

**Pull request closed without merging**